src/components/booking/SelectSessionList/SelectSessionList.tsx

```diff
@@ -11,19 +11,20 @@ import SkeletonSelectSessionCard from "./SkeletonSessionCard";
 interface SelectSessionListProps {
   isMember: boolean;
   onLimitReached: () => void;
+  memberMaxSessions: number;
   className?: string;
 }
 
 export function SelectSessionList({
   onLimitReached,
   isMember,
+  memberMaxSessions,
   className,
 }: SelectSessionListProps) {
   const { data, isLoading } = useCurrentGameSessions();
-  const maxSessions: number = isMember ? 2 : 1;
+  const maxSessions: number = isMember ? memberMaxSessions : 1;
```

Comment:
maybe instead of `memberMaxSessions` you could just pass `maxSessions` and you can delete line 25? (since this is already calculated in the parent component)